WellnessLiving/Wl/Business/DataModel.php

```diff
@@ -170,6 +178,15 @@ class DataModel extends WlModelAbstract
    */
   public $k_business = '0';
 
+  /**
+   * The franchisor business key. Empty if this business is the franchisor.
+   *
+   * @get result
+   * @var string
+   * @see \RsBusinessSql
```

Comment:
We don't include any mentions of DB tables in the SDK. Otherwise people ask for production DB access which is an argument I am tired of having.